test_scripts/UpdateVideoStreamingCapabilities/002_5_OnSystemCapabilityUpdated_out_of_range_minsize_maxsize_array.lua

```diff
@@ -0,0 +1,32 @@
+-- https://adc.luxoft.com/jira/browse/FORDTCN-XXXX
+
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/UpdateVideoStreamingCapabilities/common')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local notExpected = 0
+local isSubscribe = true
+local arraySize = {
+  minSize = 0,
+  maxSize = 101
+}
+
+--[[ Scenario ]]
+for parameter, value in pairs(arraySize) do
+  common.Title("Preconditions")
+  common.Step("Clean environment", common.preconditions)
+  common.Step("Set HMI Capabilities", common.setHMICapabilities)
+  common.Step("Start SDL, HMI, connect Mobile, start Session", common.start, { common.hmiDefaultCapabilities })
+  common.Step("RAI", common.registerAppWOPTU)
+  common.Step("Activate App", common.activateApp)
+  common.Step("GetSystemCapability with subscribe = true", common.getSystemCapability, { isSubscribe })
+
+  common.Title("Test")
+  common.Step("OnSystemCapabilityUpdated out of range " .. parameter .. " " .. value,
```

Comment:
Propose to not execute all steps for different values from `arraySize` but only current with sending of the notification

Reply:
Done (see 43db3b7)